cibuildwheel/macos.py

```diff
@@ -270,7 +270,7 @@ def setup_python(
     config_is_arm64 = python_configuration.identifier.endswith("arm64")
     config_is_universal2 = python_configuration.identifier.endswith("universal2")
 
-    if python_configuration.version == "3.9":
+    if python_configuration.version in {"3.9", "3.10"}:
```

Comment:
Should we just make this
```suggestion
    if float(python_configuration.version) >= 3.9:
```
? Or change it for an inverse not so that it doesn't grow?

Reply:
I'd rather not rely on float here.
Either cast to a packaging.Version and compare that or, as you suggest, reverse to `not in {"3.6", "3.7", "3.8"}`

Reply:
mayeut's suggestion (`not in`) looks good to me, also.